corehq/apps/reports/templates/reports/reportdata/case_details.html

```diff
@@ -7,74 +7,32 @@
 
 {% block head %} {{ block.super }}
     {% include "imports/proptable.html" %}
-    <style type="text/css">
-        .dataTables_length select {
-            display: inline-block;
-        }
-    </style>
-{% endblock %}
-
-{% block js-inline %}{{ block.super }}
-    <script>
-    $(function() {
```

Comment:
This whole section appears to not be in use; I couldn't find any references to `form_lookups` or to the classes `xform_ref` and `case_ref`. Dug in git a bit but didn't find where they were removed.